SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
+===
+
+The bzz protocol consists of a single message type; handshake.
```

Comment:
:? Also I'd format this as `handshake` if it has a specific meaning (message type)

Reply:
Yes, this is confusing perhaps. The suite of swarm protocols are (currently) divided into `bzz` `hive` `pss` `stream`. The `bzz` one is _only_ the handshake. But maybe we should rename `bzz` to `handshake` to make it more intuitive, and use `bzz` as an alias to the suite of protocols?